modules/node/src/conditionalCommitment/conditionalCommitment.entity.ts

```diff
@@ -2,31 +2,35 @@ import { OneToOne, JoinColumn, Entity, Column, PrimaryColumn } from "typeorm";
 
 import { AppInstance } from "../appInstance/appInstance.entity";
 import { IsEthAddress, IsKeccak256Hash } from "../validate";
+import { HexString } from "@connext/types";
```

Comment:
Imports don't need to be sorted but let's at least keep external imports separate from local ones